shim.js

```diff
@@ -21,7 +20,7 @@ module.exports = function shimName() {
 			configurable: true,
 			enumerable: false,
 			get: function () {
-				var name = fnCall(polyfill, this);
+				var name = bind.call(polyfill, this).call(this);
```

Comment:
The code in master is, because `fnCall` is bound at require time. `fnCall` calls its first argument, with its second and later arguments as args to the first argument. so `fnCall(this)` invokes the function - `fnCall(polyfill, this)` invokes `polyfill` with its receiver as the current function.